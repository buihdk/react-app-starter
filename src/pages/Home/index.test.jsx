import React from 'react';
import { shallow } from 'enzyme';
import I18n from 'i18n-js';

import HomePage from '.';

const tSpy = jest
  .spyOn(I18n, 't')
  .mockReturnValueOnce('mock translated title')
  .mockReturnValueOnce('mock translated description');

describe('HomePage', () => {
  test('renders without crashing', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
    expect(tSpy).toBeCalledTimes(2);
  });
});
