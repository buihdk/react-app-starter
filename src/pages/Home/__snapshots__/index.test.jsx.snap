// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders without crashing 1`] = `
<Fragment>
  <h1>
    mock translated title
  </h1>
  <p>
    mock translated description
  </p>
  <Link
    to="/mock-users"
  >
    Mock Users Page
  </Link>
  <PacMan />
</Fragment>
`;
